plugins/modules/rpm_sync.py

```diff
@@ -55,16 +64,42 @@
 """
 
 
+import re
+
 from ansible_collections.pulp.squeezer.plugins.module_utils.pulp import (
     PulpAnsibleModule,
     PulpRpmRemote,
     PulpRpmRepository,
+    SqueezerException,
 )
 
 
+def _parse_version(version_str):
```

Comment:
would using `distutils.LooseVersion` help here?

Reply:
I think these days packaging.version.parse is preferred as distutils was never documented. However when I tried that I got complaints from the linter about the module import. I guess it's not part of the standard library.